### PR TITLE
Add new tutorial: Pox virus genome analysis

### PR DESCRIPTION
**Pull request by wm75**

### CODE REVIEW

**Comment by wm75** [comment]
@TKlingstrom you're welcome to review and suggest in which form you want to be acknowledged in this one.

**Comment by TKlingstrom** [comment]
Perfect, I will look through it tonight and test it.!

File: topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,944 @@
...
+  
+Recently, a tiled-amplicon approach with separate sequencing of half-genomes has been developed for *capripoxviruses* ({% cite Mathijs2022 %}) that makes it possible to assign ITR-derived reads unambiguously to the 5'- or 3'-end of a pox virus genome when combined with a suitable bioinformatics pipeline.
+  
+This tutorial demonstrates how such data can be analyzed correctly with Galaxy.
+  
+
+> <comment-title>Advanced tutorial</comment-title>
+> The analysis presented here demonstrates a very close to real-world analysis of viral sequencing data with Galaxy. In fact, it is modeled after this [fully production-ready workflow](https://workflowhub.eu/workflows/439) for analysis of the same type of data.
+> As such, this tutorial avoids many simplifactions that are often made in training material to ease understanding.
```
Comment (shiltemann) [inline topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md:66]:
```suggestion
> As such, this tutorial avoids many simplifications that are often made in training material to ease understanding.
```

File: topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,944 @@
...
+>
+>    > <question-title></question-title>
+>    >
+>    > Has consensus building worked well?
+>    >
+>    > > <solution-title>Answer</solution-title>
+>    > >
+>    > > The tool reports only two stretches consisting of Ns at both ends of the genome, and their positions are exactly identical between the samples. There are no interior Ns, nor other ambiguity characters.
+>    > >
+>    > > The 5'-end strech of Ns extends up to position 264 (in bed format, so zero-based and the position itself not included). The binding site of the first primer in the primer scheme is from (bed format again) 235 up to 264.
```
Comment (shiltemann) [inline topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md:748]:
```suggestion
>    > > The 5'-end stretch of Ns extends up to position 264 (in bed format, so zero-based and the position itself not included). The binding site of the first primer in the primer scheme is from (bed format again) 235 up to 264.
```

**Review by shiltemann — APPROVED** [review]
looks great, thanks @wm75!

File: topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,944 @@
...
+  -
+    type: "internal"
+    topic_name: sequence-analysis
+    tutorials:
+      - quality-control
+      - mapping
+contributors:
+- wm75
+tags:
+- virology
```
Comment (hexylena) [inline topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md:40]:
```suggestion
- virology
- public-health
```

we'll make a public-health synthetic topic soon

Comment (hexylena) [reply topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md:40]: Hmm, yeah, ok. We can do that. I have an association of "one health" with a specific project (OHP) but if it's a general enough term? then yea we can go with that.

Comment (wm75) [reply topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md:40]: https://www.who.int/health-topics/one-health - it's almost turning into a buzzword these days :)

Comment (hexylena) [reply topics/variant-analysis/tutorials/pox-tiled-amplicon/tutorial.md:40]: aye, fine for me. Updated the suggestion :)

**Comment by wm75** [comment]
Thanks for spotting my typos @shiltemann :)

**Comment by wm75** [comment]
Have heard back from @TKlingstrom offline over the weekend. Going to add funding information.

File: CONTRIBUTORS.yaml [hunk 4]
```diff
@@ -1482,11 +1482,11 @@ tsenapathi:
     name: Tharindu Senapathi
     joined: 2019-05
 
-tklingstrom:
+TKlingstrom:
```
Comment (hexylena) [inline CONTRIBUTORS.yaml:1485]: shouldn't be necessary, github user names aren't case sensitive, but fine either way. 

if you're changing this, please also update topics/teaching/tutorials/experiences/2018-11-02-tklingstrom.md

**Review by hexylena — APPROVED** [review]

**Pull request merged**